dist/AppImageBuilder.yml

```
@@ -2,13 +2,14 @@ version: 1
 
 script:
 - |
-  icon_path="{{APPIMAGE_PATH}}/usr/share/icons/hicolor/128x128/apps"
+  mv "{{APPIMAGE_SOURCE}}" "${TARGET_APPDIR}"
+  icon_path="${TARGET_APPDIR}/usr/share/icons/hicolor/128x128/apps"
   mkdir -p "${icon_path}"
   cp -a dist/evdevhook2.png "${icon_path}"
+  # mksquashfs is required until fix of https://github.com/AppImageCrafters/build-appimage/issues/5
```

```suggestion
  # Manual installation is required until fix of https://github.com/AppImageCrafters/build-appimage/issues/5
```

Since problem isn't that `mksquashfs` is required, the problem is that the image is missing it.